akka-actor/src/main/scala/akka/io/dns/DnsResourceRecords.scala

```diff
@@ -29,8 +29,8 @@ sealed abstract class ResourceRecord(val name: String, val ttl: Int, val recType
 }
 
 @ApiMayChange
-final case class ARecord(override val name: String, override val ttl: Int,
-                         ip: InetAddress) extends ResourceRecord(name, ttl, RecordType.A.code, RecordClass.IN.code) {
+final case class ARecord(override val name: String, override val ttlInSeconds: Int,
```

Comment:
What is the reason we are using `Int` and not `FiniteDuration`?

Reply:
IDK, @chbatey?

Sounds like a reasonable change, though. Fixing...

Reply:
... on second thought, given the impact in amount of code changed I think we can do it on a separate [PR](https://github.com/akka/akka/pull/25850).